### PR TITLE
Fix network ci/allinone

### PR DESCRIPTION
**Pull request by ddaskal**

### CODE REVIEW

**Comment by ddaskal** [comment]
rebuild

File: roles/openstack-network/handlers/main.yml [hunk 1]
```diff
@@ -2,3 +2,14 @@
 - name: ifup neutron external interface
   command: ifup {{ neutron_external_interface }}
   ignore_errors: yes
+
+- name: enable ip forwarding
+  sysctl: name="net.ipv4.conf.all.forwarding" value=1 sysctl_set=yes state=present reload=yes
+ 
+- name: enable ip masq
```
Comment (scoop206) [inline roles/openstack-network/handlers/main.yml]: I don't think we want this here?   AFAIK the masquerade should only be necessary to support CI so can just be in playbooks/ci-full-centos/tasks/pre-deploy.yml

Comment (ddaskal) [inline roles/openstack-network/handlers/main.yml]: This is necessary for allinone deployments as well, which is why we kept it in a general network role.

**Comment by ddaskal** [comment]
Ansible 2.1.4.0 change notes says "Added a version check for jinja2-2.9, which does not fully work with Ansible currently.", pinning Ansible back to 2.1.3.0 will unblock people until upstream Ansible works again with Jinja.

**Comment by paulczar** [comment]
centos branch is now pinned so you should be able to rebase and retest.

**Comment by ddaskal** [comment]
retest

**Comment by ddaskal** [comment]
retest

**Comment by ddaskal** [comment]
retest

**Comment by ddaskal** [comment]
retest

**Comment by ddaskal** [comment]
retest

**Comment by ddaskal** [comment]
retest

**Comment by nirajdp76** [comment]
ok to test

**Pull request merged**